src/App.tsx

```diff
@@ -1,16 +1,60 @@
 import React from 'react';
 import './App.scss';
 
-class App extends React.Component {
-  addOne = () => {};
+interface State {
+  count: number,
+}
+
+class App extends React.Component<{}, State> {
+  state = {
+    count: 0,
+  };
 
-  add100 = () => {};
+  addOne = () => {
+    this.setState((prevState) => {
+      return {
+        count: prevState.count + 1,
+      };
+    });
```

Comment:
You can simplify this
```suggestion
    this.setState((prevState) => ({
        count: prevState.count + 1,
      }));
```